packages/api/models/granules.js

```diff
@@ -63,7 +63,10 @@ const renameProperty = (from, to, obj) => {
 };
 
 class Granule extends Manager {
-  constructor() {
+  constructor({
```

Comment:
This is solely to allow for mocking from the unit tests to avoid unnecessary error logs, though I think it's a nice improvement regardless